readme.md

```diff
@@ -13,20 +13,24 @@ the square brackets (for multiple-choice questions).
 
 ### Question #1
 
-Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back. (Assume the server responds with a JSON array of objects representing gifs.)
+Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back.
+(Assume the server responds with a JSON array of objects representing gifs.)
 
 Your Answer:
 ```js
-
+jQuery.ajax( "http://kittengifs.com/gifs" [results] )
+console.log(results)
 ```
```

Comment:
There's a lot more that goes into a AJAX request (e.g., url, method). And since it's a jQuery method, we initialize NOT with but `jQuery.ajax` but with `$.ajax`. Included a sample response below. Please let me know if you would like any help with AJAX. It's a key concept that we will be making heavy use of in the remainder of class.

```js
$.ajax({
  url: "http://kittengifs.com/gifs"
  method: "get"
})
  .then(function(results){
    console.log(results.length)
  });
```